src/NHibernate/Cfg/Loquacious/IDbIntegrationConfigurationProperties.cs

```diff
@@ -41,5 +41,8 @@ public interface IDbIntegrationConfigurationProperties
 		SchemaAutoAction SchemaAction { set; }
 
 		void QueryModelRewriterFactory<TFactory>() where TFactory : IQueryModelRewriterFactory;
+
+		//TODO 6.0: Uncomment. Or even better consider dropping all configuration interfaces and use classes directly 
+		//MultiTenancy.MultiTenancyStrategy MultiTenancy { set; }
```

Comment:
Is this even the right place to add? I'm a bit lost in those configuration classes.